spec/models/ability_spec.rb

```diff
@@ -190,12 +191,18 @@
       it{ should_not be_able_to(:manage, other_event.commercials.first) }
       it{ should be_able_to(:index, my_event.comment_threads.first) }
       it{ should_not be_able_to(:index, other_event.comment_threads.first) }
+
+      it{ should be_able_to(:toggle_user, cfp_role) }
+      it{ should be_able_to(:update, cfp_role) }
```

Comment:
To reduce the lines of code as we will be testing the same thing for multiple roles. Choose whichever way you prefer.